### PR TITLE
developed the new website under the name "thebull"

### PR DESCRIPTION
**Pull request by sandipswain**
Closes # 

## ✅ Checks

- [✅] My pull request adheres to the code style of this project
- [✅] My code requires changes to the documentation
- [✅] I have updated the documentation as required
- [✅] All the tests have passed

### CODE REVIEW

**Review by rodhisz — APPROVED** [review]
cool

File: public/websites/thebull/index.html [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+    </div>
+
+    <!-- Section 7 Footer -->
+    <footer class="row">
+        <div class="col-sm-4"></div>
+        <div class="col-sm-6 foot_text">
+            Designed and developed by Sandip Swain
+        </div>
+        <div class="col-sm-2 icons">
+            <img src="../../images/mail.svg" alt="mail">
```
Comment (ImBIOS) [inline public/websites/thebull/index.html:171]: don't add file outside your web folder

**Review by ImBIOS — CHANGES_REQUESTED** [review]
Please replace or remove these files

**Review by ImBIOS — APPROVED** [review]
LGTM!

**Pull request merged**